Source/WebKit/UIProcess/ios/WKScrollView.mm

```diff
@@ -555,11 +555,36 @@ - (CGPoint)_puic_contentOffsetForCrownInputSequencerOffset:(double)sequencerOffs
 #endif // HAVE(PEPPER_UI_CORE)
 
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
-- (bool)_updateOverlayRegions:(const Vector<CGRect> &)overlayRegions
+static void addDebugOverlays(CALayer *layer, const Vector<CGRect>& overlayRegions)
```

Comment:
I would probably land this in a separate patch; keeping the behavior change and the debugging UI separate.